docs/common/help.go

```diff
@@ -21,6 +21,19 @@ const GlobalEnvVars string = `	JFROG_CLI_LOG_LEVEL
 		[Default: The operating system's temp directory]
 		Defines the temp directory used by JFrog CLI.
 
+	JFROG_CLI_BUILD_NAME
+		Build name to use in build related commands. This environment variable will be used in case of omitted --build-name flag and <build name> argument.
+	
+	JFROG_CLI_BUILD_NUMBER
+		Build number to use in build related commands. This environment variable will be used in case of omitted '--build-number' flag and '<build number>' argument.
+
+	JFROG_CLI_BUILD_URL
+		Can be used for setting the CI server build URL in the build-info. This environment variable will be used in case of omitted '--build-url' flag.
+	
+	JFROG_CLI_ENV_EXCLUDE
+		[Default: *password*;*secret*;*key*;*token*] 
+		List of case insensitive patterns in the form of "value1;value2;...". Environment variables match those patterns will be excluded. This environment variable will be used in case of omitted '--env-exclude' flag.
```

Comment:
...This environment variable will be used in case of omitted '--env-exclude' flag. -->
...This environment variable is used by the "jfrog rt build-publish" command,  in case the --env-exclude command option is not sent.